util/crypto/mnemonic.go

```diff
@@ -6,12 +6,11 @@ import (
 	"errors"
 	"fmt"
 
+	"github.com/anyproto/any-sync/util/go-ethereum/accounts"
```

Comment:
Can we name this just ethereum and not go-ethereum/accounts?

And also let's remove unused functions

Reply:
Also pls make imports correct - divided in 3 parts: go-libs, other packages, any-sync packages. This is not done everywhere, but we started doing this at some point

Reply:
it should be done by the linter automatically like in MW. Ok I'll do it